### ISSUE TITLE
Send output to a build directory

### ISSUE THREAD

**Issue by texasflood** [vim-latex/vim-latex#18]
(Enhancement)

An option to make sure all output files go to a build directory would be great.

**Comment by gerw** [comment]
Yes, this would be a good idea. I do not compile with the latex-suite, therefore I will not implement it. Pull requests are welcome.

**Comment by alick** [comment]
I guess this can be implemented by binding the shortcut key of compilation (`\ll`) to latexmk with the `-outdir` option (or `$outdir` in `.latexmkrc`), without changing vim-latex itself. No tests though.
